Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -23,121 +23,91 @@ import Shared
+        static let ctaButtonWidth: CGFloat = 350
+        static let ctaButtonWidthSmall: CGFloat = 300
+        static let ctaButtonBottomSpace: CGFloat = 60
+        static let ctaButtonBottomSpaceSmall: CGFloat = 5
+        static let closeButtonSize = CGRect(width: 44, height: 44)
+    }
 
+    // MARK: - Properties
     var onViewDismissed: (() -> Void)?
+
     // Public constants
     let viewModel = DefaultBrowserOnboardingViewModel()
     let theme = LegacyThemeManager.instance
 
     // Private vars
-    private var fxTextThemeColour: UIColor {
-        // For dark theme we want to show light colours and for light we want to show dark colours
-        return theme.currentName == .dark ? .white : .black
-    }
-
-    private var fxBackgroundThemeColour: UIColor = UIColor.theme.onboarding.backgroundColor
-
-    private var descriptionFontSize: CGFloat {
-        return screenSize.height > 1000 ? DBOnboardingUX.fontSizeXSmall :
-               screenSize.height > 668 ? DBOnboardingUX.fontSize :
-               screenSize.height > 640 ? DBOnboardingUX.fontSizeSmall : DBOnboardingUX.fontSizeXSmall
-    }
-
     private var titleFontSize: CGFloat {
-        return screenSize.height > 1000 ? DBOnboardingUX.titleSizeLarge :
-               screenSize.height > 640 ? DBOnboardingUX.titleSize : DBOnboardingUX.titleSizeSmall
+        return screenSize.height > 1000 ? UX.titleSizeLarge :
+               screenSize.height > 640 ? UX.titleSize : UX.titleSizeSmall
     }
 
     // Orientation independent screen size
     private let screenSize = DeviceInfo.screenSizeOrientationIndependent()
 
     // UI
-    private lazy var closeButton: UIButton = .build { [weak self] button in
-        button.setImage(UIImage(named: "close-large"), for: .normal)
-        button.tintColor = .secondaryLabel
-        button.addTarget(self, action: #selector(self?.dismissAnimated), for: .touchUpInside)
+    private lazy var scrollView: UIScrollView = .build { view in
+        view.backgroundColor = .clear
     }
 
-    private let textView: UIView = .build { view in }
+    private lazy var containerView: UIView = .build { _ in }
+
+    private lazy var closeButton: UIButton = .build { button in
+        button.setImage(UIImage(named: ImageIdentifiers.closeLargeButton), for: .normal)
+        button.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.HomeTabBanner.closeButton
+    }
 
     private lazy var titleLabel: UILabel = .build { [weak self] label in
-        guard let self = self else { return }
-        label.text = self.viewModel.model?.titleText
-        label.font = .boldSystemFont(ofSize: self.titleFontSize)
+        label.font = DynamicFontHelper.defaultHelper.preferredBoldFont(withTextStyle: .title1,
+                                                                       size: self?.titleFontSize ?? UX.titleSize)
         label.textAlignment = .center
-        label.numberOfLines = 2
+        label.numberOfLines = 0
```

Comment:
Number of lines changed here, that's expected in this PR? Same thing for descriptionText.

Reply:
I removed it the restrictions of lines as well as the max font size as we now have a scroll view to fit all of the content. What good does it do to show instructions on how to set the default browser if the text might be cut off?